sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -293,6 +296,13 @@ func deleteCtrIfExists(ctx context.Context, client *containerd.Client, targetCtr
 	return nil
 }
 
+func withMountLabel(label string) oci.SpecOpts {
+	return func(_ context.Context, _ oci.Client, _ *containers.Container, s *runtimespec.Spec) error {
+		s.Linux.MountLabel = label
```

Comment:
nit: Check for a nil `s.Linux` before writing.

```suggestion
		if s.Linux != nil {
			s.Linux.MountLabel = label
		}
```